CondTools/SiPixel/test/run_0T_BoR3.sh

```diff
@@ -0,0 +1,2 @@
+cmsRun SiPixelTemplateDBObjectUploader_cfg.py Fullname=SiPixelTemplateDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=../data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/mapping.csv TemplateFilePath=CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1
```

Comment:
@tvami maybe this can become a unit test?

Reply:
https://github.com/cms-sw/cmssw/commit/bd78af91aa10ffba3af0bade8b4959c0a7baada1